en/chromebook_setup/instructions.md

```diff
@@ -55,7 +55,7 @@ mkdir djangogirls
 cd djangogirls
 python3.6 -mvenv myvenv
 source myvenv/bin/activate
-pip install django~=1.11.0
+pip install django
```

Comment:
Would it make sense to still install a specific version?

Reply:
Agreed, even if this makes the tutorial up to date, it still needs to pin to the version this tutorial was updated for in case